testing/unit/renderer/specs/store/PersistentState.spec.js

```diff
@@ -0,0 +1,36 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import App from '@/App.vue';
+import persistentState from '@/store/storage/PersistentState';
+import projectInfo from '@/store/modules/ProjectInfo';
+
+describe('PersistentState.js', () => {
+  Vue.use(Vuex);
```

Comment:
Yeah, now that I'm seeing it all over the place, maybe this should live in `App.vue`.

Reply:
This just sets up the testing environment up with the Vuex plugin. The plugin needs to be installed before anything attempts to use `Vue.store` otherwise we might run into problems. Adding this line to `App.vue` would result in the `Vuex.Store` being created and passed to Vue before the plugin gets installed.